azure-automation-plus-dsc-lab/dsc/xActiveDirectory/Assert-HADC.ps1

```diff
@@ -0,0 +1,91 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingComputerNameHardcoded', '')]
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+param()
+
+# A configuration to Create High Availability Domain Controller
+$secpasswd = ConvertTo-SecureString "Adrumble@6" -AsPlainText -Force
+$domainCred = New-Object System.Management.Automation.PSCredential ("sva-dscdom\Administrator", $secpasswd)
+$safemodeAdministratorCred = New-Object System.Management.Automation.PSCredential ("sva-dscdom\Administrator", $secpasswd)
+$localcred = New-Object System.Management.Automation.PSCredential ("Administrator", $secpasswd)
+$redmondCred = Get-Credential -Message "Enter Credentials for DNS Delegation: "
+$newpasswd = ConvertTo-SecureString "Adrumble@7" -AsPlainText -Force
```

Comment:
remove hardcoded password

Reply:
I didn't author this script. It appears to just be part of the unit test for the xActiveDirectory module and this credential isn't hardcoded in the azuredeploy.parameters.json file anyway. I'm not sure if changing the Assert-HADC.ps1 script will break the testing functionality for the module.